src/pages/book/publish.js

```diff
@@ -0,0 +1,172 @@
+import PostBook from "../../components/book/PostBook";
+// Util
+import ScreamSkeleton from "../../util/ScreamSkeleton";
+// MUI Core
+import Grid from "@material-ui/core/Grid";
+import withStyles from "@material-ui/core/styles/withStyles";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogActions from "@material-ui/core/DialogActions";
+// Redux
+import { connect } from "react-redux";
+import { getBookData } from "../../redux/actions/dataActions";
+import axios from "axios";
+const styles = (theme) => ({
+  ...theme.spread,
+  inProgress: {
+    marginBottom: "5em",
+  },
+  inProgressTitle: {
+    color: "#1c2a48",
+    fontWeight: "bolder",
+    fontFamily: "Times New Roman",
+    textShadow: "2px 2px #fff",
+  },
+  bookCardContainer: {
+    display: "flex",
+  },
+  publishBody: {
+    display: "flex",
+  },
+  dialogTitle: {
+    "& span": {
+      color: "blue",
+    },
+  },
+  avatar: {
+    margin: theme.spacing(1),
+    marginLeft: theme.spacing(2),
+    backgroundColor: theme.palette.primary.main,
+  },
+});
+class bookreview extends Component {
+  state = {
+    screamIdParam: null,
+    open: false,
+    bookTitle: null,
+  };
+  handleOpen = (e) => {
+    this.setState({ open: true });
+    this.setState({
+      bookTitle: e.currentTarget.lastChild.textContent,
+    });
+    console.log(e.currentTarget.lastChild.textContent);
+  };
+  handleClose = () => {
+    this.setState({ open: false });
+  };
+  componentDidMount() {
+    const handle = this.props.match.params.handle;
+    const screamId = this.props.match.params.screamId;
+
+    if (screamId) this.setState({ screamIdParam: screamId });
+
+    this.props.getBookData(handle);
+    axios
+      .get(`/user/${handle}`)
+      .then((res) => {
+        this.setState({
+          profile: res.data.user,
+        });
+      })
+      .catch((err) => console.log(err));
+  }
+  render() {
+    const { screams, loading } = this.props.data;
+    const { classes } = this.props;
+    const { screamIdParam } = this.state;
+    let recentScreamsMarkup = loading ? (
+      <ScreamSkeleton />
+    ) : screams === null ? (
+      <p>No screams from this user</p>
+    ) : !screamIdParam ? (
+      screams.map((scream) => (
+        <BookCard
+          onClick={this.handleOpen}
+          key={scream.screamId}
+          scream={scream}
+        />
+      ))
+    ) : (
+      screams.map((scream) => {
+        if (scream.screamId !== screamIdParam)
+          return (
+            <BookCard
+              onClick={this.handleOpen}
+              key={scream.screamId}
+              scream={scream}
+              div_name={scream.createdAt}
+            />
+          );
+        else
+          return <BookCard key={scream.screamId} scream={scream} openDialog />;
+      })
+    );
+    let confirmDialog = (
```

Comment:
you don't have an image on the side like the ticket described, Is that ok?

Reply:
> Functionality works as described! again just some styling issues you should ask someone about?

I will talk about that with daughter who is taking art in her school.

Reply:
> you don't have an image on the side like the ticket described, Is that ok?

Can you explain more please? I can not draw picture on my brain. so sorry for that

Reply:
https://proudtale.atlassian.net/browse/PT-52?jql=order%20by%20created%20DESC

Reply:
I'm talking about this

Reply:
> https://proudtale.atlassian.net/browse/PT-52?jql=order%20by%20created%20DESC

Now, I see. I will add proudtale image  on left side on the pop up. Thank you.